### ISSUE TITLE
Accessibility: Missing Status message for screen reader users updating cart in shortcode and cart classic Block

### ISSUE THREAD

**Issue by amberhinds** [woothemes/woocommerce#58121]
### Prerequisites

- [x] I have carried out troubleshooting steps and I believe I have found a bug.
- [x] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

This is a follow up on https://github.com/woocommerce/woocommerce/issues/51902.

In the shortcode version of the cart, changing the quantity field for a product does not save the change. There is an "Update Cart" button that is added below the table, which must be clicked to save changes and truly change the quantity. This is action does not have a screen reader success message confirming that the quantity is changed.

See the video recording here: https://github.com/woocommerce/woocommerce/issues/51902#issuecomment-2867394585

Component or Page:
Shortcode: [woocommerce_cart] & Cart Classic Block

Name affected element:
Updating cart after changing the product quantity.

### Expected behavior

When updating the cart, screen readers will announce the update action was successful.

### Actual behavior

Screan readers will read the "Cart Updated" message after the "Update Cart" button is pressed.

### Steps to reproduce

1. Configure WooCommerce to use the [woocommerce_cart] shortcode.
2. Turn on VoiceOver or NVDA screen reader.
3. Add a product to the cart.
4. Go to the cart page.
5. On the cart page increase the product quantity to a larger number.
6. An update cart button will appear below the products table.
7. Tab to the update cart button.
8. Press return or spacebar to trigger the update cart button.
9. Observe that the quantity change is saved but the screen reader does not announce the save success.

### WordPress Environment

N/A

### Isolating the problem

- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/products/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**Comment by opr** [comment]
@kdo hi I see you assigned this to yourself. What is the context behind that, are you working on a fix? If so, what is the progress there, will a PR be open soon?

**Comment by vikrampm1** [comment]
Hi @opr yes this will be worked on by @kdo and we will have a PR up soon. Once it's gone through our internal review/QA we will request Woo review.

**Comment by vikrampm1** [comment]
@opr I see you've also self-assigned here, are you planning to tackle this?

**Comment by opr** [comment]
No, please continue :)
